### PR TITLE
Update nwbextractors.py

### PR DESCRIPTION
**Pull request by bendichter**
use latest tooling for streaming data

### CODE REVIEW

**Review by h-mayorquin — APPROVED** [review]
Sounds good for me.

File: src/spikeinterface/extractors/nwbextractors.py [hunk 1]
```diff
@@ -498,11 +498,11 @@ class NwbRecordingExtractor(BaseRecording, _BaseNWBExtractor):
     >>>
     >>> # get s3 path
     >>> dandiset_id, filepath = "101116", "sub-001/sub-001_ecephys.nwb"
-    >>> with DandiAPIClient("https://api-staging.dandiarchive.org/api") as client:
```
Comment (bendichter) [inline src/spikeinterface/extractors/nwbextractors.py:501]: oh, actually we may need to keep this, since I guess the sample data is in staging

Comment (h-mayorquin) [reply src/spikeinterface/extractors/nwbextractors.py:501]: I guess we coudl jsut select other dataset that is not in staging area which I think would be better. What do you think?

Comment (alejoe91) [reply src/spikeinterface/extractors/nwbextractors.py:501]: @h-mayorquin so should we pick another dataset?

Comment (h-mayorquin) [reply src/spikeinterface/extractors/nwbextractors.py:501]: Yes.I think it will be better if the docstring does not have that complication.

**Review by alejoe91 — APPROVED** [review]

**Pull request merged**